pygeoapi/util.py

```diff
@@ -164,6 +171,41 @@ def to_json(dict_, pretty=False):
                       indent=indent)
 
 
+def format_datetime(value, format_=DATETIME_FORMAT):
+    """
+    Parse datetime as ISO 8601 string; re-present it in particular format
+    for display in HTML
+
+    :param value: `str` of ISO datetime
+    :param format_: `str` of datetime format for strftime
+
+    :returns: string
+    """
+
+    if not isinstance(value, str) or not value.strip():
+        return ''
+
+    print("VALUE", value)
```

Comment:
Remove

Reply:
Just line 188, not sure why the rest is highlighted

Reply:
Fixed